src/core/query-builder.utils.ts

```diff
@@ -0,0 +1,76 @@
+import { QueryBuilderCustomOperation } from "smart-webcomponents-react";
+import { QueryBuilderOption } from "./types";
+import { ComputedDataFields } from "./query-builder.constants";
+
+/**
+ * Removes extra queries from computed fields so the query builder will read only one
+ * @param query actual query that has been sent
```

Comment:
Since this is a method using a regex, it'd be great to give a small example in the description of this method.